packages/create-plugin/templates/_partials/backend-getting-started.md

```diff
@@ -1,6 +1,6 @@
 ### Backend
 
-1. Update [Grafana plugin SDK for Go](https://grafana.com/developers/plugin-tools/introduction/grafana-plugin-sdk-for-go) dependency to the latest minor version:
+1. Update [Grafana plugin SDK for Go](../backend/grafana-plugin-sdk-for-go) dependency to the latest minor version:
```

Comment:
```suggestion
1. Update [Grafana plugin SDK for Go](https://grafana.com/developers/plugin-tools/key-concepts/backend-plugins/grafana-plugin-sdk-for-go) dependency to the latest minor version:
```

This isnt used in the website, its used in the generated templates - we cant use relative links